### PR TITLE
Flow operator defines memory requests/limits for containers (#33)

### PR DESCRIPTION
**Pull request by pnedelko**

### CODE REVIEW

File: services/flows-operator/src/FlowOperator.js [hunk 1]
```diff
@@ -58,24 +58,24 @@ class FlowOperator {
                 await this._undeployJob({
                     metadata: {
                         name: node.metadata.name
-                    } 
-                });  
+                    }
+                });
```
Comment (anton-kotenko) [inline services/flows-operator/src/FlowOperator.js:62]: plz add rule for line ending into eslintrc

File: services/flows-operator/src/FlowOperator.js [hunk 2]
```diff
@@ -256,20 +256,39 @@ class FlowOperator {
...
+                          resources: this._prepareResourcesDefinition(flowModel, node)
                         }]
                     }
                 }
             }
         };
     }
 
+    _prepareResourcesDefinition(flow, node) {
+        const cpuLimit = node.cpuLimit || flow.cpuLimit || this._config.get('DEFAULT_CPU_LIMIT');
```
Comment (anton-kotenko) [inline services/flows-operator/src/FlowOperator.js]: Plz update Readme.md with this additions to flow definition format

File: services/flows-operator/src/FlowOperator.js [hunk 3]
```diff
@@ -256,20 +256,39 @@ class FlowOperator {
...
                         restartPolicy: 'Never',
                         containers: [{
-                            image: node.image,
-                            name: 'apprunner',
-                            imagePullPolicy: 'Always',
-                            env: Object.keys(envVars).map(key => ({
-                                name: key,
-                                value: envVars[key]
-                            }))
+                          image: node.image,
```
Comment (anton-kotenko) [inline services/flows-operator/src/FlowOperator.js]: AFAIK  we are using 4-spaces indent. Why only two here?

**Review by anton-kotenko — CHANGES_REQUESTED** [review]
Generally looks good, but few minor issues

File: services/flows-operator/src/FlowOperator.js (hunk 3 above)
Comment (pnedelko) [reply services/flows-operator/src/FlowOperator.js]: sorry, my IDE was misconfigured

File: services/flows-operator/src/FlowOperator.js (hunk 1 above)
Comment (pnedelko) [reply services/flows-operator/src/FlowOperator.js:62]: @anton-kotenko sorry, what rule do you mean?

**Comment by pnedelko** [comment]
@falkvoigt could you please help with the build here?

**Comment by falkvoigt** [comment]
there is no Problem the cloud billing needs to be activated. That is already adressed and is done tomorrow hopefully.

**Comment by falkvoigt** [comment]
I dont get why this is building? Do you miss the changes from anton? Or are you pushing on master in OIH project?

**Comment by anton-kotenko** [comment]
Seems I've found source of problem for this case why deploy and build runs even though it's pull request.
We have next conditional expression in .travis.yml for build and deploy steps
`branch = master AND fork = false`

According to docs for travis https://docs.travis-ci.com/user/conditional-builds-stages-jobs/
```
# require the branch name to be master (note for PRs this is the base branch name)
if: branch = master
```



Maybe we may add next expression to condition  `AND pull_request = false`
(see https://docs.travis-ci.com/user/conditions-v1) 
@pnedelko  could you check please?

**Review by anton-kotenko — APPROVED** [review]

**Pull request merged**